src/app/verification-addresses/features/iva-manager-list/iva-manager-list.component.html

```diff
@@ -0,0 +1,62 @@
+@if (ivasAreLoading()) {
+  <p class="text-base">Loading your contact addresses..</p>
+} @else {
+  @let columns = ['user', 'type', 'value', 'state', 'actions'];
+  <table
+    mat-table
+    [dataSource]="source"
+    class="mat-elevation-z8"
+    matSort
+    #sort="matSort"
+  >
+    <ng-container matColumnDef="user">
+      <th mat-header-cell *matHeaderCellDef class="w-64" mat-sort-header>User</th>
+      <td mat-cell class="text-base" *matCellDef="let iva">
+        {{ iva.user_name }}
+      </td>
+    </ng-container>
+    <ng-container matColumnDef="type">
+      <th mat-header-cell *matHeaderCellDef class="w-15" mat-sort-header>Type</th>
+      <td mat-cell class="text-base" *matCellDef="let iva">
+        <mat-icon>{{ typeIconName(iva) }}</mat-icon
```

Comment:
These need some styling so they look less separated from the text, but I can deal with that and other minor improvements in another PR.

Reply:
Thanks, would be great if you could polish this and the colors in a separate PR, so we can merge already.